wkcuber/README.md

```diff
@@ -159,25 +53,6 @@ poetry run pylint -j4 wkcuber
 poetry run pytest tests
 ```
 
-Please, run the extended test suite:
-```bash
-tests/scripts/all_tests.sh
-```
-
-PyPi releases are automatically pushed when creating a new Git tag/Github release. 
-
-## API documentation
-Check out the [latest version of the API documentation](https://docs.webknossos.org/api/wkcuber.html).
-
-### Generate the API documentation
-Run `docs/generate.sh` to open a server displaying the API docs. `docs/generate.sh --persist` persists the html to `docs/api`.
-
-## Test Data Credits
```

Comment:
I guess, we still use this testdata.

Reply:
Test data credits are now from webknossos readme.